Projects/1-Beginner/Calculator-App/try.py

```
@@ -0,0 +1,30 @@
+    print(word)
+    sortedword = ""
+    for i in word:
+        sortedword = sortedword + i
+    return sortedword
+
+def ranks(arr):
+    res = []
+    for i in range(len(arr)):
+        res.append(1)
+        for j in range(len(arr)):
+            if (arr[i] > arr[j]) == False and arr[i] != arr[j]:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not arr[i] > arr[j]:` for false checks

Replace with `not arr[i] > arr[j]`

(E712)